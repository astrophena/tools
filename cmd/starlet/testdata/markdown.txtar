update: start
-- bot.star --
load("@starlet//tg.star", "tg")

def handle(update):
    tg.send_message(config.owner_id, "# Hello, world!\nI'm **Ilya**!")
    # Should be an error (invalid arguments).
    markdown.convert()
