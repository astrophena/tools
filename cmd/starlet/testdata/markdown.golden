[
  {
    "method": "sendMessage",
    "args": {
      "chat_id": 123456789,
      "entities": [
        {
          "language": "",
          "length": 14,
          "offset": 0,
          "type": "bold",
          "url": ""
        },
        {
          "language": "",
          "length": 4,
          "offset": 19,
          "type": "bold",
          "url": ""
        }
      ],
      "reply_markup": {},
      "text": "Hello, world!\n\nIâ€™m Ilya!\n\n"
    }
  }
]